### ISSUE TITLE
Better errored job reports

### ISSUE THREAD

**Issue by amercader** [frictionlessdata/goodtables.io#214]
These are not useful at all:

![klnqnfy](https://cloud.githubusercontent.com/assets/200230/25906711/4c9cbd16-359d-11e7-84a2-9a760d9d3e79.png)

Perhaps we can not show the original exception (in case it leaks some information about the server) but at least we should explain whether something went wrong on our side, configuration provided was incorrect, etc

- [ ] Show error type (Wrong configuration, error during validation, etc)
- [ ] @roll Can we catch the error on a particular file and carry on with the other ones?

**Comment by roll** [comment]
@amercader 
About catching error on particular file:
- for many cases it's already will be just a e.g. format error etc for only one file inside the report
- for invalid table schema for now it will be an internal error - blocked by https://github.com/frictionlessdata/goodtables-py/issues/174
- for invalid datapackage for now it will be an internal error - waiting for specv-v1 tabular resource

So I think we could start with proper messages for errors happened on `goodtables.io` side.

**Comment by amercader** [comment]
@roll but for cases like the causanatura-datos repo it just doesn't make sense that just because one file out of 50 fails (because of a bug in goodtables-py) the whole report is unavailable. I just want to know if that is supported on the goodtables-py side (carry on with validation if a file fails).

**Comment by roll** [comment]
@amercader 
It's point number 1 - it should be a report with 1 file having `format-error`. So it's a goodtables bug (fixing it now)

**Comment by CharlesNepote** [comment]
I've got it also but don't what to do...

http://goodtables.io/github/CharlesNepote/CSV-test-files

Thanks for the work already done.
Charles.

**Comment by amercader** [comment]
@CharlesNepote We should be pushing fixes soon that will make the errors more meaningful, thanks!
